README.md

```diff
@@ -39,6 +39,15 @@ saucectl run --test-env sauce
 
 ![sauce cloud example](assets/sauce_cloud_example.gif)
 
+### In Sauce Cloud with iOS Simulators
+# TODO (change gif file to ios + sauce mode)
+
+```shell
+saucectl run -c .sauce/config_ios.yml --test-env sauce
```

Comment:
This `--test-env` is now deprecated, it'd be better to update the README and config accordingly 💡

Reply:
@tianfeng92 has already created a PR to remove usage of `--test-end. We need to merge her changes firstly and after that, I can update my PR.